chef_postgres/templates/pg_hba_conf.erb

```diff
@@ -87,16 +87,17 @@ local   all             postgres                                trust
 # TYPE  DATABASE        USER            ADDRESS                 METHOD
 
 # "local" is for Unix domain socket connections only
-local   all             all                                     md5
+local   all             all                                     trust
 # IPv4 local connections:
-host    all             all             127.0.0.1/32            md5
+host    all             all             127.0.0.1/32            trust
 # IPv4 remote connections - change this line if you want to limit access to specific IP ranges.
-host    all             all             0.0.0.0/0               md5 
+host    all             all             0.0.0.0/0               trust
```

Comment:
We shouldn't have the trust line for public connections.  Only md5

Reply:
that was removed.